src/components/Header/Header.css

```diff
@@ -1,56 +1,328 @@
+    bottom: 140px;
+    left: -11px;
+  }
+  25%, 75% {
+    transform: rotate(90deg);
+    bottom: 166px;
+    left: 14px;
+  }
+  50% {
+    bottom: 166px;
+    left: 125px;
+  }
+}
+
+.header__left-column {
+  position: relative;
+  display: flex;
+}
+
+.header__left-column :first-child {
+  position: absolute;
+  width: 4px;
+  height: 184px;
+  background-color: #fff;
+  bottom: -10px;
+  right: 100px;
+}
+
+.header__left-column :first-child:before {
+  content: '';
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  bottom: 0;
+  right: -18px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: straight-Y 32s linear infinite;
+}
+
+.header__left-column :nth-child(2) {
+  position: absolute;
+  width: 182px;
+  height: 45px;
+  bottom: 95px;
+  right: 135px;
+  transform: scaleX(-1);
+}
+
+.header__left-column :nth-child(2):before {
+  content: '';
+  width: 26px;
+  height: 26px;
+  position: absolute;
+  bottom: 30px;
+  left: 156px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: left-curve-X 18s linear infinite;
+}
+
+.header__left-column :nth-child(3) {
+  position: absolute;
+  width: 45px;
+  height: 182px;
+  bottom: 170px;
+  right: 135px;
+  transform: rotate(-180deg);
+}
+
+.header__left-column :nth-child(3):before {
+  content: '';
+  width: 32px;
+  height: 32px;
+  position: absolute;
+  bottom: 0;
+  left: -14px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: left-curve-Y 15s linear infinite;
 }
 
+.header__right-column {
+  position: relative;
+}
+
+.header__right-column :first-child {
+  position: absolute;
+  width: 4px;
+  height: 184px;
+  background-color: #fff;
+  bottom: 70px;
+  left: 120px;
+}
+
+.header__right-column :first-child:before {
+  content: '';
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  bottom: 0;
+  right: -18px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: straight-Y 12s linear infinite;
+}
+
+.header__right-column :nth-child(2) {
+  position: absolute;
+  width: 150px;
+  height: 182px;
+  bottom: 10px;
+  left: 165px;
+}
+
+.header__right-column :nth-child(2):before {
+  content: '';
+  width: 26px;
+  height: 26px;
+  position: absolute;
+  bottom: 0;
+  left: -11px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: right-curve-Y 18s linear infinite;
+}
+
+.header__right-column :nth-child(3) {
+  position: absolute;
+  width: 150px;
+  height: 90px;
+  bottom: 207px;
+  left: 165px;
+  transform: scaleY(-1);
+}
+
+.header__right-column :nth-child(3):before {
+  content: '';
+  width: 20px;
+  height: 20px;
+  position: absolute;
+  bottom: 78px;
+  left: 130px;
+  background-color: #faa805;
+  z-index: 1;
+  animation: right-curve-X 18s linear infinite;
+}
+
+@media screen and (max-width: 1070px) {
+  .header {
+    padding: 150px 40px;
+  }
 
+  .header__left-column{
+      transform: scale(0.5);
+  }
 
+  .header__right-column{
+    transform: scale(0.5);
+}
+}
+
+@media screen and (max-width: 700px) {
```

Comment:
i believe per the designs we should have breakpoints at 1280, 768, and 320 and without the last one it seems we don't have mobile responsiveness? could we perhaps add those or adjust current 1070 & 700 to match the first two and add a 320?

Reply:
Hey @hotcheetoqueen , try changing the grid declaration in header from `3fr` to `1fr 1fr 1fr`. It looks almost perfect then.
@TanyaKomar is that what you intented? 

After making that change the only issue I see with responsiveness is between 700 and 767 the layout becomes too cramped. I think you can probably just bump the media query at 700 up to 767.
![image](https://user-images.githubusercontent.com/37159177/112403379-af839380-8ce4-11eb-8004-c19885d925ea.png)

Reply:
hmm, i'm sure it's fine if it looks fine, but stylistically when many devs are sharing a repo its often good to create consistent standards, like set breakpoints. no biggie though!

Reply:
Yeah, `3fr` is a mistake. I have `1fr 330px 1fr` for header grid.
@hotcheetoqueen for smooth transition it is not necessary to have breakpoints exactly at 1280, 768, and 320.